api-server/Web/Controllers/GitHubController.cs

```diff
@@ -1894,8 +1912,10 @@ public async Task<ActionResult> FilterMergedPRs([FromBody] PRFilter filter)
         using (NpgsqlConnection connection = new NpgsqlConnection(_coreConfiguration.DbConnectionString))
         {
             await connection.OpenAsync();
+            for (int batch = 0; batch < totalBatches; batch++)
+            {
 
-            string selects = "pullid, title, pullnumber, author, authoravatarurl, createdat, updatedat, reponame, additions, deletions, changedfiles, comments, labels, repoowner, checks, checks_complete, checks_incomplete, checks_success, checks_fail, assignees, reviews, reviewers";
+                string selects = "pullid, title, pullnumber, author, authoravatarurl, createdat, updatedat, reponame, additions, deletions, changedfiles, comments, labels, repoowner, checks, checks_complete, checks_incomplete, checks_success, checks_fail, assignees, reviews, reviewers";
```

Comment:
<!--Using HubReview-->
nitpick(non-blocking): There seems to be an extra space on this line.